libs/chrome-ext/src/lib/options/options.html

```diff
@@ -1,34 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>Nuclia options</title>
-  <link rel="stylesheet" href="options.css">
+  <link rel="stylesheet"
+        href="../style.css">
 </head>
 
 <body>
-  <p>Set the Knowledge Box where you want to upload web content:</p>
+  <main>
+
+    <fieldset>
+      <legend>Set the Knowledge Box where you want to upload web content</legend>
 
-  <div class="field">
-    <label for="kb">Knowledge box ID:</label>
-    <input id="kb" type="text" autocomplete="off">
-  </div>
+      <div class="field">
+        <label for="kb">Knowledge box ID:</label>
+        <input id="kb"
+               type="text"
+               autocomplete="off">
+      </div>
 
-  <div class="field">
-    <label for="zone">Zone ID:</label>
-    <input id="zone" type="text" autocomplete="off">
-  </div>
+      <div class="field">
+        <label for="zone">Zone ID:</label>
+        <input id="zone"
+               type="text"
+               autocomplete="off">
+      </div>
 
-  <div class="field">
-    <label for="key">API Key:</label>
-    <input id="key" type="text" autocomplete="off">
-    <div class="help">Enter an API key with <strong>Contributor</strong> role</div>
-  </div>
+      <div class="field">
+        <label for="key">API Key:</label>
+        <input id="key"
+               type="text"
+               autocomplete="off">
+        <div class="help">Enter an API key with <strong>Contributor</strong> role</div>
+      </div>
+    </fieldset>
+    <fieldset>
+      <legend>YouTube</legend>
+      <div class="field">
+        <label for="youTubeKey">YouTube API key:</label>
+        <input id="youTubeKey"
+               type="text"
+               autocomplete="off">
+        <div class="help">Enter a YouTube Data API</div>
+      </div>
+    </fieldset>
```

Comment:
is this new fieldset required for all use case? if not, should we display something explaining when you need it?